e2e/ansible/playbooks/feature/data-integrity-fio/demo-fio-read-openebs.yml

```diff
@@ -0,0 +1,49 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: basic-read
+data:
+
+ basic-rw : |-
+
+     [global]
+     directory=/datadir
+
+     [basic-fio]
+     rw=read
```

Comment:
@nsathyaseelan , can you confirm if we are reading the "same" file as written in the write job ? or does this job "rewrite" a file and read it ? (I presume it is the former, but just wanted to confirm)

Reply:
@ksatchit verified if it is same file by matching md5sum